a troubleshooting info page

salsakran opened the pull request:
@senior this is the code we paired on but in a working end-to-end state.

The actual clj api endpoint is stubbed out, but in the format the frontend needs.

@kdoh I could use some help making this look a little less like I wrote it.

salsakran committed:
some basic scaffolding for the troubleshooting info page

salsakran commented on frontend/src/metabase/admin/TroubleshootingApp.jsx:
```diff
@@ -0,0 +1,49 @@
+    render () {
+        return (
+            <LoadingAndErrorWrapper loading={!this.props.troubleShootingInfo}>
+                { this.props.troubleShootingInfo && 
```
The fact that this isn't taken care of by the LoadingAndErrorWrapper makes me peeved. If you don't explicitly guard against nulls, you get null pointer errors.

kdoh replied:
I think you need to have the child of LoadingAndErrorWrapper be a function. If I understand properly, LAEW calls it's child function when things have "loaded" and that closure prevents the issue you're hitting.
```
{ () => (
 ... your stuff here
}
```

kdoh replied:
(It should definitely do a better job of letting you know that's what you need to do though.)

salsakran committed:
pass lint

iethree committed:
Add db timezone info to a util API response

This adds a new protocol function to IDriver that will return a
DateTime (timezone preserved) and return that to be included in a
timezone debugging page.

salsakran committed:
include redux store size as well

salsakran committed:
style this a bit

salsakran committed:
include browser timezone info

camsaul commented:
@salsakran @senior are we still thinking about merging this?

camsaul commented:
@salsakran @senior bump

salsakran commented:
@kdoh this was a different page. This is for things like timezone information, and any info on the jvm. This should be resurrected and styled

camsaul commented:
we have troubleshooting info pages now, if we want to add this stuff we can open a new PR because this one is so out-of-date
